datadog_checks_dev/datadog_checks/dev/tooling/git.py

```diff
@@ -68,7 +68,7 @@ def files_changed(include_uncommitted=True):
     return sorted([f for f in set(changed_files) if f])
 
 
-def get_commits_since(check_name, target_tag=None, exclude_branch=None):
+def get_commits_since(check_name, target_tag=None, end='', exclude_branch=None):
```

Comment:
`end` is usually passed in as `None`, so we get failures when not using the flag:
`Captured Output: fatal: ambiguous argument 'http_check-6.1.2...None': unknown revision or path not in the working tree.`